ambry-server/src/integration-test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -539,6 +553,101 @@ private static void testLatePutRequest(BlobId blobId, BlobProperties properties,
     assertEquals(expectedErrorCode, response.getError());
   }
 
+  /**
+   * Tests blobs put to dataNode can be backed up by {@link CloudBackupManager}.
+   * @param cluster the {@link MockCluster} of dataNodes.
+   * @param dataNode the datanode where blobs are originally put.
+   * @param clientSSLConfig the {@link SSLConfig}.
+   * @param clientSSLSocketFactory the {@link SSLSocketFactory}.
+   * @param testEncryption if encryption will be tested. Not used now.
+   * @param notificationSystem the {@link MockNotificationSystem} to track blobs event in {@link MockCluster}.
+   */
+  static void endToEndBackupManagerTest(MockCluster cluster, DataNodeId dataNode, SSLConfig clientSSLConfig,
+      SSLSocketFactory clientSSLSocketFactory, boolean testEncryption, MockNotificationSystem notificationSystem)
+      throws Exception {
+    // TODO: test encryption
+    int blobBackupCount = 10;
+    ClusterMap clusterMap = cluster.getClusterMap();
+    // Send blobs to DataNode
+    byte[] userMetadata = new byte[100];
```

Comment:
put these `100` integer literals into a static private variable

Reply:
Create local vars for these size. Because other tests in this file use different integers.